README.md

```
@@ -1,142 +1,147 @@
+[2.](#dependencies)<a id="dependencies"></a> **Install all dependencies** :beginner: :package:
+
+  ```sh
+  $ yarn install
+  ```
+
+## <a id="usage"></a>Usage <a href="#toc" title="Table of Contents"><kbd>⮭</kbd></a>
+
+[1.](#watch) <a id="watch"></a>**Start watching** for file changes :eyes:
+
+  ```sh
+  $ yarn run watch # yarn start
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)